www-client/seamonkey/seamonkey-2.53.18.2-r1.ebuild

```diff
@@ -0,0 +1,568 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_AUTOCONF="2.1"
+
+PYTHON_COMPAT=( python3_{10..11} )
```

Comment:
... and no newer python support? :)

Reply:
No, python 3.12 isn't supported yet, build system calls deprecated functions, which are removed in 3.12. I'm working with upstream to get this in.